bindings/pydeck/examples/path_layer.py

```diff
@@ -1,13 +1,20 @@
+"""
+PathLayer
+=========
+
+Locations of the Bay Area Rapid Transit lines.
+"""
+
```

Comment:
consistency nit: Just noticing i see different levels of whitespace between these comments and the imports.